db/seeds.rb

```diff
@@ -5,11 +5,103 @@
+Movement.create!(identity_id: identity_2.id, direction: "departure", carrier_date_time: "2017-09-22 13:00:00")
+Movement.create!(identity_id: identity_2.id, direction: "arrival", carrier_date_time: "2017-08-28 23:45:00")
+Movement.create!(identity_id: identity_2.id, direction: "departure", carrier_date_time: "2017-11-06 09:30:00")
+Movement.create!(identity_id: identity_2.id, direction: "arrival", carrier_date_time: "2017-11-11 23:25:00")
+Movement.create!(identity_id: identity_2.id, direction: "departure", carrier_date_time: "2017-11-29 09:00:00")
+Movement.create!(identity_id: identity_2.id, direction: "arrival", carrier_date_time: "2017-11-30 16:30:00")
+Movement.create!(identity_id: identity_2.id, direction: "departure", carrier_date_time: "2018-04-28 05:50:00")
+Movement.create!(identity_id: identity_2.id, direction: "arrival", carrier_date_time: "2018-04-29 23:30:00")
+Movement.create!(identity_id: identity_2.id, direction: "departure", carrier_date_time: "2018-07-03 08:50:00")
+Movement.create!(identity_id: identity_2.id, direction: "arrival", carrier_date_time: "2018-07-06 23:20:00")
+Movement.create!(identity_id: identity_2.id, direction: "departure", carrier_date_time: "2018-09-28 06:40:00")
+Movement.create!(identity_id: identity_2.id, direction: "arrival", carrier_date_time: "2018-10-03 23:45:00")
+
+#########################
+
+# James Jones
+# Gender change from Male to Female
+
+country_3 = Country.create!(name: "Australia")
+client_2 = Client.create!(im_client_id: "123456", file_number: "2")
+identity_3 = Identity.create!(client_id: client_2.id, identity_number: "Y-4183346-E", family_name: "Jones", first_name: "James", second_name: "", third_name: "", gender: "Male", country_of_birth_id: country_3.id, nationality: country_3.name, issuing_state_id: country_3.id, serial_number: "Y-4775136-E")
+identity_4 = Identity.create!(client_id: client_2.id, identity_number: "Y-5183346-E", family_name: "Jones", first_name: "Jaymie", second_name: "", third_name: "", gender: "Female", country_of_birth_id: country_3.id, nationality: country_3.name, issuing_state_id: country_3.id, serial_number: "Y-5775136-E")
+Movement.create!(identity_id: identity_3.id, direction: "arrival", carrier_date_time: 100.days.ago)
+Movement.create!(identity_id: identity_3.id, direction: "departure", carrier_date_time: 90.days.ago)
+Movement.create!(identity_id: identity_3.id, direction: "arrival", carrier_date_time: 80.days.ago)
+Movement.create!(identity_id: identity_3.id, direction: "departure", carrier_date_time: 70.days.ago)
+
+Movement.create!(identity_id: identity_4.id, direction: "arrival", carrier_date_time: 60.days.ago)
+Movement.create!(identity_id: identity_4.id, direction: "departure", carrier_date_time: 50.days.ago)
+Movement.create!(identity_id: identity_4.id, direction: "arrival", carrier_date_time: 40.days.ago)
+Movement.create!(identity_id: identity_4.id, direction: "departure", carrier_date_time: 30.days.ago)
```

Comment:
For the final version, we'll want to change this to `find_or_create_by` so that it's not adding these every time the server runs seeds (which is potentially every merge), but it should be fine for the presentation.

Reply:
@danieljauch Thanks, I'll change it now so we are all good.